includes/steps/class-step-update-user.php

```diff
@@ -302,12 +302,16 @@ public function process() {
 
 		if ( ! $user ) {
 			$this->log_debug( __METHOD__ . '(): user not found. Bailing.' );
+			/* translators: The step name */
```

Comment:
The comment usually contains the placeholder.
```suggestion
			/* translators: s%: The step name */
```
Source https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#variables